.scrutinizer.yml

```diff
@@ -0,0 +1,118 @@
+filter:
+    paths:
+        - 'htdocs/*'
+    excluded_paths:
+        - 'bin/*'
+        - 'doc/*'
+        - 'local/*'
+    dependency_paths:
+        - 'htdocs/vendor/*'
+
+checks:
+    php:
+        fix_php_opening_tag: false
+        remove_php_closing_tag: false
+        avoid_closing_tag: true
+        one_class_per_file: true
+        side_effects_or_types: true
+        no_mixed_inline_html: false
+        require_braces_around_control_structures: false
+        php5_style_constructor: true
+        no_global_keyword: true
```

Comment:
Siehe zwei drüber - das gäbe im Moment tonnenweise Violations, braucht ein grundlegendes Refactoring.